pkg/reconciler/taskrun/taskrun_test.go

```diff
@@ -1326,6 +1326,17 @@ func TestReconcile_SetsStartTime(t *testing.T) {
 	}
 	testAssets, cancel := getTaskRunController(t, d)
 	defer cancel()
+	clients := testAssets.Clients
+
+	t.Logf("Creating SA %s in %s", "default", "foo")
+	if _, err := clients.Kube.CoreV1().ServiceAccounts("foo").Create(&corev1.ServiceAccount{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "default",
+			Namespace: "foo",
+		},
+	}); err != nil {
+		t.Fatal(err)
+	}
```

Comment:
Why is this needed ? :thinking:

Reply:
If you remove it the test will fail with missing service account.
It looks like returning the error exposes some hidden issues in existing tests.

Reply:
ah ok :sweat_smile: